Card/src/main/java/com/paypal/android/card/Card.kt

```
@@ -1,7 +1,7 @@
 package com.paypal.android.card
 
 data class Card(
-    val cardNumber: String,
-    val expirationDate: String,
-    val securityCode: String
+    var number: String = "",
+    var expirationDate: String = "",
```

It looks to me we're expecting merchants to pass in a string of format `MM/YY` correct? 

In iOS we're having a `CardExpiry` class that takes in a `month: Int` and `year: Int` => then internally `CardExpiry` will be responsible for formatting the date to format `YYYY-MM` that our server expects. What do you think of doing that here? :smile:

---

And what do you think on naming `expirationDate` vs `expiry`?
we're doing `expiry` like in the request body but either works for me

---

Yeah we did something similar with `ExpiryDate`. I think `expirationDate` will probably read better for merchants, but technically either could work.

---

yep I think `expirationDate` is clearer to merchants too. I can change `CardExpiry` -> `ExpirationDate` in iOS.

Question: Are we able to use the new `CardExpiry` here instead of `String`? 
And also what do you think of `CardExpiry(val month: Int, val year: Int)` vs `CardExpiry(val month: String, val year: String)` ?

---

Yeah I'm down with storing month and year as a string. I guess I envisioned this class as being internal though. The best developer experience would probably be for the merchant to provide a string IMO.

---

But there are some concerns. We would probably need to surface an error if the merchant were to provide something in an incorrect format.

---

I see you just changed it to Int [here](https://github.com/paypal/Android-SDK/commit/0b885993afc4e37f79e95f7c8cce625b667abd71#diff-9376f8fdbbaec31b90530d944ffcf2ee3211b4e44753261bd81b1023149532e4R3). I think it looks great, we're having the same `ExpirationDate(month: Int, year: Int)` in iOS :smile:

---

just wondering should the line `var expirationDate: String` above be changed to `var expirationDate: ExpirationDate` instead?